Documentation/migration-scripts/python-scripts/repush_user_kafka_messages.py

```
@@ -0,0 +1,321 @@
+            name = row['name']
+            username = row['username']
+            tenant = row['tenant_code']
+            created_at = row['created_at']
+            updated_at = row['updated_at']
+            status = row['status']
+            meta_json = row['meta']
+            
+            if isinstance(meta_json, str):
+                try:
+                    meta_json = json.loads(meta_json)
+                except:
```

Ruff (0.15.1)

[error] Do not use bare `except`

(E722)